src/index.ts

```diff
@@ -7,4 +7,5 @@ export * as onchain_event_types from './lib/runtime/onchain_events';
 export * as providers from './providers';
 export * as crypto_hash from './crypto_hash';
 export * as encoding from './encoding';
+export * as decode_event from './onchain-events/decode-event';
```

Comment:
change
onchain-events/decode-event
to
onchain_events/decode_event

new folder/files name should keep the same name pattern with the older ones.